ui/imports/shared/popups/ImageCropWorkflow.qml

```diff
@@ -26,6 +27,11 @@ Item {
         fileDialog.open()
     }
 
+    function cropImage(imageUrl) {
```

Comment:
This function lets you skip the file dialog right? If so, then my comment on the fixtures_root is irrelevant. I might copy you for my PR in the end.

Reply:
aaha yes, its does skip the filedialog completely!